Extensions/quick_tags.js

```diff
@@ -86,9 +83,14 @@ XKit.extensions.quick_tags = new Object({
 		$(document).on('click', '.xkit-tag-add', XKit.extensions.quick_tags.add_button_clicked);
 
 		XKit.interface.post_window_listener.add("quick_tags", XKit.extensions.quick_tags.post_window);
-		XKit.post_listener.add("quick_tags", XKit.extensions.quick_tags.do_posts);
-		this.do_posts();
 
+		// don't run this if we're on the new post iframe
+		if ($("[data-id]").length && XKit.page.react) {
```

Comment:
the new post iframe isn't react-based, and tumblr (at least previously) doesn't always load with posts, so this should just be a check for react
```suggestion
		if (XKit.page.react) {
```